app/model/command/Allocation.scala

```diff
@@ -22,14 +22,13 @@ import model.exchange.AssessorSkill
 import play.api.libs.json.{ Json, OFormat }
 
 trait Allocation {
-  def id: String
   def status: AllocationStatus
 }
 
 case class AssessorAllocation(
-  id: String,
-  status: AllocationStatus,
-  allocatedAs: AssessorSkill
+                               userId: String,
+                               status: AllocationStatus,
+                               allocatedAs: AssessorSkill
```

Comment:
https://stackoverflow.com/questions/26880677/intellij-scala-plugins-case-class-indentation-is-absurd